settings.php

```diff
@@ -102,5 +102,17 @@
         $settings->add(new admin_setting_configtext('tool_securityquestions/lockoutnum',
                     get_string('settingslockoutnum', 'tool_securityquestions'),
                     get_string('settingslockoutnumdesc', 'tool_securityquestions'), 3, PARAM_INT));
+
```

Comment:
add heading with extra paragraph of examples

change defaults to be off so we have feature parity with previous version